tests/driver_bme280/main.c

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2016 Kees Bakker, SODAQ
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the BME280 temperature, pressure
+ *              and humidity sensor.
+ *
+ * @author      Kees Bakker <kees@sodaq.com>
+ *
+ * @}
+ */
+
+#ifndef TEST_I2C
+#error "TEST_I2C not defined"
+#endif
+#ifndef TEST_I2C_ADDR
+#error "TEST_I2C_ADDR not defined"
+#endif
+/* These defines to pass the above values on to bme280_params.h */
+#define BME280_PARAM_I2C_DEV    TEST_I2C
+#define BME280_PARAM_I2C_ADDR   TEST_I2C_ADDR
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "bme280_params.h"
```

Comment:
in the contrary. This file is all it takes, just get rid of all the `TEST_` defines.